src/services/API/RootClient.js

```diff
@@ -90,6 +91,11 @@ client.interceptors.response.use(
         .catch((error) => console.log(error));
     }
 
+    if (status === 404) {
+      router.replace({ name: "404" });
+      return Promise.reject(error);
+    }
+
```

Comment:
this should be part of the ErrorHandling thing made below right?

Reply:
And only the router part can be added there. The promise.reject is anyways happening at the end of the function